### DISCUSSION TITLE
Send to user without phone number nor username?

### DISCUSSION THREAD

**Discussion by orazioedoardo** [AsamK/signal-cli#1485] [Q&A] [answered]
Now that usernames and phone number privacy are live, in the app it's possible to send a message to a user without knowing a phone number nor username, provided that the sender has a connection with the recipient, like sharing a group or the recipient having sent a message to the sender first, thus creating a session. Is there a way to do this with signal-cli?

**Comment by AsamK** [comment]
You can send a message to the user's uuid (ACI).

**Reply by orazioedoardo** [reply]
Thanks, it worked! I just sent a message using the UUID as recipient to the send command. Now I'm wondering whether this would work even if the sender has no relationship with the recipient and viceversa other than knowing it's ACI.

**Reply by AsamK** [reply, marked as answer]
Yes, it would work, you only need to know the ACI.
That's how usernames work, signal-cli asks the Signal servers for the ACI of a given username and then sends a message to that ACI.
